examples/platform/silabs/uart.cpp

```diff
@@ -411,7 +411,7 @@ static void UART_rx_callback(UARTDRV_Handle_t handle, Ecode_t transferStatus, ui
 
 #ifdef ENABLE_CHIP_SHELL
     chip::NotifyShellProcess();
-#elif !defined(PW_RPC_ENABLED)
+#elif !defined(PW_RPC_ENABLED) && CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI
```

Comment:
It would make sense to apply a similar change at line 367 and replace !defined(SL_WIFI) with CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI

Reply:
gotcha ! done.